flake.nix

```diff
@@ -0,0 +1,152 @@
+{
+  description = "plutus-simple-model";
+
+  inputs = {
+    haskell-nix.url = "github:L-as/haskell.nix/ac825b91c202947ec59b1a477003564cc018fcec";
+    haskell-nix.inputs.nixpkgs.follows = "haskell-nix/nixpkgs-unstable";
+
+    nixpkgs.follows = "haskell-nix/nixpkgs-2105";
```

Comment:
Use `unstable` or `2111`. `2105` is ancient.

Reply:
Considering what you do above, you should just do `nixpkgs.follows = "haskell-nix/nixpkgs";`